src/analytics/events.js

```diff
@@ -0,0 +1,5 @@
+export const INSERT_BLOCK = 'insertBlock';
```

Comment:
Tracking additional Redux actions starts here. First export the action name as a const and add to the appropriate Redux namespace below.